bots/learn-trigger

```diff
@@ -45,7 +45,7 @@ def main():
         "tests-data " + TRAINING_DATA,
         # We want this to run in both namespaces
         "learn-tests http://cockpit-learn.cockpit.svc.cluster.local/train/" + TRAINING_DATA,
```

Comment:
Is that URL still a thing then?

Reply:
so this should be the url for the centosci clutser, this won't work if the task starts on bos. However we'd need to expose the webdav part of the server which is a pain. But the alternative is always hoping the task would run on centosci which is also a pain.